packages/api-reference/src/features/ExampleRequest/ExampleRequest.vue

```diff
@@ -348,7 +348,7 @@ function updateHttpClient(value: string) {
   padding-left: 12px;
   padding-right: 9px;
 }
-.request-card-footer {
+.request-card-footer.request-card-footer {
```

Comment:
Due to some reordering of how the styles are injected this needed more specificity.

Reply:
Should we leave a comment? 🤔